data/data.json

```diff
@@ -2610,17 +2610,12 @@
    "year": 1980
  },
  {
+   "quote": "Is it just me, or is it getting crazier out there?",
     "quote": "Bright day uh, Bright Boy?",
```

Comment:
The model here is not correct as there are 2 *quote* fields
+ The quote "Bright day uh, Bright Boy?" is already @ line 1749, one of them should be deleted

Reply:
ohhh, sorry didn't notice, I fix it, check now please.